loaders/mock.js

```diff
@@ -5,25 +5,30 @@ const events = require('../data/default/events.json')
 const competitors = require('../data/default/competitors.json')
 const { logger } = require('../services/helper')
 
-async function resetLocalData (app) {
-  AWS.config.update({ region: app.get('awsRegion') })
-  _addEventsToCompetitors(events, competitors)
+function resetLocalData (app) {
+  return new Promise(async (resolve, reject) => {
+    let ecUrl
+    AWS.config.update({ region: app.get('awsRegion') })
+    _addEventsToCompetitors(events, competitors)
 
-  try {
-    const obj = { id: '1', competitors, events }
-    await _uploadToDynamo(process.env.DYNAMO_TABLE, obj)
-  } catch (err) {
-    logger.error(`Failed to connect to DynamoDB with ${err.code}`)
-  }
+    try {
+      const obj = { id: '1', competitors, events }
+      ecUrl = await _uploadToDynamo(process.env.DYNAMO_TABLE, obj)
```

Comment:
Is this variable really the elastiCache url? Seems like it's just the response from the upload.

Reply:
yes, this function will overwrite only the competitors and the events object. since we are modifying the same table, it will not touch the `ecUrl`. This is needed because when we do reset with EC enabled, the server is reset, so it will reset the DB also

in that case, we may only want to reset the `event` and `competitors` table on initial setup and not elsewhere